### DISCUSSION TITLE
Does aphrodite-engine support models quantized with vllm's llm-compressor?

### DISCUSSION THREAD

**Discussion by BlairSadewitz** [PygmalionAI/aphrodite-engine#563] [Q&A] [answered]
For example, could I use it to quantize a model with smoothquant to do 8-bit lossless inference later?

**Comment by AlpinDale** [comment, marked as answer]
Not at the moment, no. But I'm currently backporting all the missing vllm features in rc_054 branch. I'm currently about 2 months behind so it'll take a week or so. In the meantime, I've added support for 4, 6, 8, and 12 bits from deepspeedfp in that branch. A later update will add support for 5, 6, 7, and a better 8bit, with good performance at higher batch sizes. You can use deepspeedfp by specifying `-q deepspeedfp --num-deepspeedfp-bits {4,6,8,12}` when launching a 16bit model. Disclaimer: its rather slow.

**Comment by AlpinDale** [comment]
There's some support for llm-compressor models in the `rc_054` branch now, but may not be completely up-to-date. I'm adding more of their quant schemes as I go through the library.

**Reply by BlairSadewitz** [reply]
Didn't see you answered this, sorry.  Yeah, I had used llm-compressor a few days ago, but it had some issues. 

BTW, does the EETQ support work?
